test/functional/mempool_resurrect.py

```diff
@@ -52,7 +52,7 @@ def run_test(self):
         for node in self.nodes:
             node.invalidateblock(blocks[0])
 
-        # mempool should be empty, all txns confirmed
+        # All txns should be back in mempool with 0 confirmations
```

Comment:
Could remove the comment, since it is mentioned two lines up already?

Reply:
Removed the first occurrence instead, to put the information closer to where it's relevant.